warehouse/query-core/src/main/java/datawave/query/config/ShardQueryConfiguration.java

```diff
@@ -753,6 +758,7 @@ public void copyFrom(ShardQueryConfiguration other) {
         this.setUseQueryTreeScanHintRules(other.isUseQueryTreeScanHintRules());
         this.setQueryTreeScanHintRules(other.getQueryTreeScanHintRules());
         this.setFieldIndexHoleMinThreshold(other.getFieldIndexHoleMinThreshold());
+        this.setIndexExpansionHintKey(other.getIndexExpansionHintKey());
```

Comment:
I thought we were going to configure the ScannerFactory, probably just adding to the hintsByTable to have entries matching some of our functions.  So we would add a configuration to that map for "expansion" or "indexExpansion".  Then the getScanner(table, function) would look in that table first by the function, and if empty then lookup by table.  I would rather keep that configuration all together instead of add another config item here.

Reply:
Ah, it appears that the table hints in the GenericQueryConfiguration does that.  I don't think we need to configure an index expansion hint key, that can be a reserved word.  We should add a comment (probably where the table hints are configured) that denotes any reserved keys are defined (e.g. QUERY_EXPANSION or something like that).